master/buildbot/test/fake/fakedb.py

```diff
@@ -2322,7 +2322,11 @@ def getMaster(self, masterid):
         return defer.succeed(None)
 
     def getMasters(self):
-        return defer.succeed(sorted(self.masters.values()))
+        def masterKey(master):
+            return master['id']
+
+        return defer.succeed(sorted(self.masters.values(),
+                                    key=masterKey))
```

Comment:
I agree this is the kind of styling issue that leads to disagrement. I think we both have reasonable arguments, I am often reluctant going to fight for it.

But we need to be consistent at least.